src/qrscan/main.py

```diff
@@ -106,9 +106,9 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         Clock.schedule_once(self._after_init)
 
-    def _after_init(self, dt):
+    def on_kv_post(self, *largs):
```

Comment:
Heads up that function was scheduled here above using `Clock.schedule_once()`.
Make sure to also clean that up and then also the `__init__()` overwrite if we no longer need it.
Test it locally for regression to be sure it's still working after the change